sdk/elixir/dev/dagger.json

```diff
@@ -2,6 +2,6 @@
   "name": "elixir-sdk-dev",
   "engineVersion": "v0.18.8",
   "sdk": {
-    "source": "elixir"
+    "source": ".."
```

Comment:
Can I change it to the local SDK? So I don't need to wait for PR to merge for dogfooding the feature.

Reply:
:thinking: yeahhhh. Don't think there's a reason we can't :thinking: 

As long as the elixir SDK doesn't rely on an unreleased version of the engine, then we're okay!